src/views/workflow-history/workflow-history-timeline-reset-button/__tests__/workflow-history-timeline-reset-button.test.tsx

```diff
@@ -0,0 +1,190 @@
+import { HttpResponse } from 'msw';
+
+import { render, screen, fireEvent } from '@/test-utils/rtl';
+
+import { type WorkflowExecutionCloseStatus } from '@/__generated__/proto-ts/uber/cadence/api/v1/WorkflowExecutionCloseStatus';
+import { type DescribeWorkflowResponse } from '@/route-handlers/describe-workflow/describe-workflow.types';
+import mockResolvedConfigValues from '@/utils/config/__fixtures__/resolved-config-values';
+import { mockResetActionConfig } from '@/views/workflow-actions/__fixtures__/workflow-actions-config';
+import { resetWorkflowActionConfig } from '@/views/workflow-actions/config/workflow-actions.config';
+import { describeWorkflowResponse } from '@/views/workflow-page/__fixtures__/describe-workflow-response';
+
+import WorkflowHistoryTimelineResetButton from '../workflow-history-timeline-reset-button';
+import type Props from '../workflow-history-timeline-reset-button.types';
+jest.mock('@/views/workflow-actions/config/workflow-actions.config', () => {
+  const originalModule = jest.requireActual(
+    '@/views/workflow-actions/config/workflow-actions.config'
+  );
+  return {
+    ...originalModule,
```

Comment:
nit: you could just do this inline
```suggestion
    ...jest.requireActual('@/views/workflow-actions/config/workflow-actions.config'),
```